src/components/Board.js

```diff
@@ -8,26 +8,123 @@ import NewCardForm from './NewCardForm';
 import CARD_DATA from '../data/card-data.json';
 
 class Board extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+
+    this.emojiLibrary = require('emoji-dictionary');
+
 
     this.state = {
       cards: [],
     };
   }
 
+  emojify = (emojiTextName) => {
+    let validEmojiCheck = this.emojiLibrary.names.includes(emojiTextName)
+    return validEmojiCheck ? this.emojiLibrary.getUnicode(emojiTextName) : emojiTextName
+  }
+
+  componentDidMount() {
+    const GET_CARDS_URL = 'https://inspiration-board.herokuapp.com/boards/mightymichelle/cards';
+
+    axios.get(GET_CARDS_URL)
+    .then((response) => {
+      this.setState({
+        cards: response.data
+      });
+    })
+    .catch((error) => {
+      this.setState({
+        error: error.message
+      });
+    });
+    //
+    //
+    //
+    // this.setState({
+    //   cards: fetchCardData()
+    // });
+  }
+
+deleteCard = (cardID) => {
+  axios.delete(`https://inspiration-board.herokuapp.com/cards/${cardID}`)
+     .then((response) => {
+       let card = this.state.cards.find((card) => card.id === response.data.card.id);
+       let updatedCardCollection = this.state.cards.filter(element => element !== card);
+
+       this.setState({cards: updatedCardCollection});
```

Comment:
Hm, weird, the list of cards isn't immediately updating for me after deleting a card... I haven't looked into debugging this though